tests/eslint-rule-tester.js

```diff
@@ -8,12 +8,17 @@ const { RuleTester } = require("eslint")
 const { FlatRuleTester } = require("eslint/use-at-your-own-risk")
```

Comment:
I am starting to think we should make these patches for different eslint versions into a package